deployment/network-operator/README.md

```diff
@@ -447,6 +447,7 @@ containerResources:
 | `ofedDriver.upgradePolicy.waitForCompletion.podSelector`    | string | not set                           | specifies a label selector for the pods to wait for completion before starting the driver upgrade                                                                             |
 | `ofedDriver.upgradePolicy.waitForCompletion.timeoutSeconds` | int    | not set                           | specify the length of time in seconds to wait before giving up for workload to finish, zero means infinite                                                                    |
 | `ofedDriver.containerResources`                             | []     | not set                           | Optional [resource requests and limits](#container-resources) for the `mofed-container` container                                                                             |
+| `ofedDriver.useOcpDriverToolkit` | bool | `true` | In OpenShift, use Driver Toolkit image to compile OFED drivers                                                                                                                                                  |
```

Comment:
prevent setting to this to true in validation webhook if we are not in openshift ?

Reply:
i now remember we discussed the flow :) so maybe we should just ignore this on non openshift

Reply:
alternative: dont introduce any API changes for this in CRD, add ENV var in operator (DISABLE_DTK or similar)

since for openshift we do want dtk to always run. in special cases where we dont, we can deploy operator with this env var.

this is best option imo as going forward we either want dtk or precompiled

Reply:
Removed from Helm